apps/challenges/views.py

```diff
@@ -1123,6 +1124,17 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
 
             zip_config.challenge = challenge
             zip_config.save()
+
+            if settings.SLACK_WEB_HOOK_URL != "http://testslackwebhook.com/webhook":
+                slack_challenge_details = {
+                    'challenge_title': challenge.title,
+                    'zip_file_link': CHALLENGE_ZIP_DOWNLOAD_LOCATION
+                }
+                send_slack_notification(
+                    message="A *new challenge* has been uploaded to EvalAI. \n *User email*: {}\n *Challenge details*: {}"
```

Comment:
`A *new challenge* has been uploaded to EvalAI. \n *User email*: {}\n *Challenge details*: {}`
`A *challenge* config is uploaded from {}. \n The challenge title is: {}`